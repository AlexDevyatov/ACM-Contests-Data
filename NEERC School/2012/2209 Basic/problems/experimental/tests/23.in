255 182
22 170
45 35
55 54
16 10
18 5
88 148
64 71
24 80
89 34
131 170
25 145
93 32
108 130
136 168
168 112
120 176
127 74
29 35
90 50
166 18
74 97
146 61
122 139
161 152
19 23
115 12
23 33
98 39
53 162
110 98
116 160
9 48
143 126
6 102
129 162
101 58
83 108
158 26
153 60
2 160
8 167
113 48
31 68
162 89
146 158
1 57
4 73
142 32
78 104
99 22
59 147
157 46
95 170
42 45
66 20
112 177
158 182
179 62
120 167
57 106
107 28
150 179
54 66
95 123
71 18
126 171
65 72
19 150
143 150
127 115
42 80
161 97
57 163
57 112
33 107
37 26
132 36
66 123
42 20
157 179
93 3
104 107
104 116
142 93
8 59
20 71
35 163
83 10
84 33
74 120
153 71
165 101
4 90
78 162
126 76
68 96
180 116
119 125
98 65
170 23
18 134
149 181
69 141
23 116
103 162
88 73
145 56
161 170
159 115
129 132
115 169
12 82
151 9
174 58
25 71
69 120
177 148
88 94
27 107
101 32
46 75
139 64
85 169
94 182
13 61
7 145
3 128
112 13
144 102
147 56
138 129
67 120
14 148
131 130
179 116
161 45
182 22
112 158
162 164
72 117
173 147
121 80
150 41
115 78
63 152
3 5
179 136
97 142
131 65
8 106
108 107
4 15
128 172
55 71
44 175
160 137
9 121
114 123
91 11
141 55
123 19
25 65
42 149
152 172
88 4
129 143
38 119
107 104
23 166
117 116
136 122
171 53
111 59
41 128
53 147
46 60
152 75
63 141
73 136
38 159
79 165
162 172
68 62
92 94
108 6
134 163
114 179
24 77
117 65
179 71
23 175
64 88
166 82
103 116
73 96
178 171
78 86
177 17
101 77
166 47
15 158
68 7
102 65
130 101
141 114
103 42
73 168
17 53
172 91
74 56
4 87
135 69
96 120
84 124
35 23
43 23
41 24
144 171
64 109
4 18
159 140
175 101
7 159
89 144
53 132
164 49
83 24
22 127
131 128
21 157
136 165
120 121
145 177
174 6
146 24
181 75
126 93
80 120
180 127
174 62
100 158
121 157
57 28
177 182
74 101
48 10
128 69
132 140
7 70
89 101
157 130
43 121
98 21
155 133
105 10
0 0 1 4 2 1 4 2 1 2 1 2 2 1 2 0 0 4 2 2 2 1 5 2 2 2 1 1 1 0 1 1 2 1 2 0 0 1 0 0 1 2 1 0 2 3 0 3 0 0 0 0 5 0 2 2 3 1 3 1 0 3 0 2 3 2 0 0 1 0 3 0 2 0 0 0 2 3 0 3 0 1 1 0 1 1 0 1 2 1 1 1 2 0 1 3 2 1 1 1 5 3 0 3 1 0 2 2 1 1 0 4 0 1 2 3 1 0 2 3 2 1 2 1 0 3 2 3 1 2 1 2 0 1 1 3 1 1 1 2 2 2 1 0 0 3 1 1 1 3 1 4 1 0 1 0 2 1 1 1 1 3 1 1 2 1 1 1 0 4 3 0 1 1 1 1 4 0 3 2 1 1
