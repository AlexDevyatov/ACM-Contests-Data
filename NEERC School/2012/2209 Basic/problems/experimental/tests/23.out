170 35 55 10 5 148 64 80 34 170 25 93 108 136 112 176 127 29 90 18 97 146 122 152 19 12 33 98 53 110 116 48 126 102 129 101 83 26 60 160 8 48 31 89 146 57 4 142 78 99 59 46 170 45 66 112 158 62 167 57 107 179 66 95 18 171 65 150 150 115 80 161 57 112 107 26 132 123 20 179 93 104 104 142 59 20 163 10 33 120 153 101 4 78 126 96 180 119 65 23 18 149 69 23 162 73 56 170 159 132 115 12 151 58 71 120 177 88 27 32 46 139 85 182 13 7 3 13 102 147 138 120 14 130 179 45 22 112 162 117 173 121 150 78 152 5 136 97 131 8 108 15 128 71 175 137 9 123 11 55 19 25 42 152 4 143 119 104 23 116 136 53 59 41 53 46 152 141 73 38 165 162 62 92 6 134 114 77 65 71 23 64 82 116 96 171 86 177 77 166 15 7 102 130 141 42 168 53 91 56 4 135 96 124 35 23 24 171 109 18 140 101 7 89 53 164 24 127 128 21 165 121 177 174 146 181 126 80 180 62 100 157 28 177 101 48 128 140 7 101 157 43 21 155 105
